doc/source/whatsnew/v0.18.1.txt

```diff
@@ -73,8 +73,7 @@ API changes
 
 - ``CParserError`` is now a ``ValueError`` instead of just an ``Exception`` (:issue:`12551`)
 
-
-
+- Using ``apply`` on resampling groupby operations now has the same output types as similar ``apply``s on other groupby operations. (:issue:`11742`, :issue:`12362`).
```

Comment:
don't list the PR number, only the issue number. give some more context, like: ``.groupby(...).apply(...)``